doc/sphinx/user/methods/stokes-solver/solver.md

```diff
@@ -0,0 +1,236 @@
+# Stokes solver
+
+
+
+Problems in ASPECT require solving the Stokes
+problem below for velocity $u$ and pressure
+$p$
+
+```{math}
+:label: eq:strong-form
+-\nabla \cdot (2 \eta(x)\varepsilon(u))+\nabla p &= f \quad \text{ in } \Omega \\
+-\nabla \cdot u&=0 \quad\text{ in } \Omega
+```
+
+equipped with boundary conditions.
+
+
+Here, $x$ lies in our
+domain $\Omega$ and $\eta(x)$ is our
+viscosity. $\varepsilon(u)=\frac{1}{2}(\nabla u+
+\nabla u^T)$ is known as the strain rate tensor.
+
+In this document, we discuss the case
+of Dirichlet boundary conditions
+$u=g_D$ on $\Gamma_D$ and Neumann
+$n \cdot (pI-2\varepsilon(u))=g_N$ on
+$\Gamma_N$. However,
+ASPECT also supports
+other boundary conditions.
```

Comment:
actually for the Stokes equation I dont think we support other boundary conditions (we support "no" boundary conditions in the sense of homogeneous Neumann boundaries).